roles/update_etc_hosts/tests/test.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: test play
+  hosts: localhost
+  remote_user: root
```

Comment:
This is not mentioned anywhere that in order to use this role you have to connect to remote host with root user. 
Connecting with root user as per my understanding is considered not secure, better to connect with ordinary user and elevate access on the host. 

What do you think about adding necesary become: true and become_user: root directives where necessary, thus limiting tasks that require root only to those that really need that kind of access. For instance ansible_check_mode clearly does not require root access, and clearly provides customer value.

Reply:
good catch. done this. but this was not necessary for testing though